config/config.yaml

```diff
@@ -1,3 +1,6 @@
+debug: false
+log_level: INFO
```

Comment:
Copied over from production.yaml

As per https://github.com/qdrant/qdrant/issues/1771#issuecomment-1518735959 we want to default to a production like configuration.

Reply:
Yes! In development mode it would simply pick up the `config/development.yaml` at runtime, fantastic.